### ISSUE TITLE
Error exporting data to MS Access

### ISSUE THREAD

**Issue by artembrizinskij** [salvois/LargeXlsx#36]
Hello!
I'm trying to create a new data source in ms access from an xlsx file that I created using LargeXlsx.
![image](https://github.com/salvois/LargeXlsx/assets/17120874/8b2564d3-17a9-4ab6-aae5-b8809ca7842f)

But I get the following error:
![image](https://github.com/salvois/LargeXlsx/assets/17120874/510a2582-f713-4f9d-9b6f-352ee6e9589d)

If I just open xlsx in ms excel and resave it (without editing), then the data source is created successfully after that.

**Comment by artembrizinskij** [comment]
I decided to look into it deeper, and it seems to have already been fixed in this commit https://github.com/salvois/LargeXlsx/commit/ade2fed09db904eb90c98e1169b9ec85038cd331
As I found out, the row number attribute was required for the export to work

**Comment by RFlipper** [comment]
@salvois  I would appreciate if you find some time to create a new new release with the fix.

**Comment by MarkPflug** [comment]
This is likely my fault: https://github.com/salvois/LargeXlsx/pull/21

According to the specification:
```XML
<xsd:complexType name="CT_Row">
 <xsd:sequence>
 <xsd:element name="c" type="CT_Cell" minOccurs="0" maxOccurs="unbounded"/>
 <xsd:element name="extLst" minOccurs="0" type="CT_ExtensionList"/>
 </xsd:sequence>
 <xsd:attribute name="r" type="xsd:unsignedInt" use="optional"/>
```

Note the "r" attribute is marked as optional, so I would argue this is a bug in MS Access.

However, I think https://github.com/salvois/LargeXlsx/pull/31 added an option to force these attributes to be written, because it seems like a lot of readers don't correctly handle this scenario. So, you should be able to produce compliant files already with slight modification to your code.

**Comment by salvois** [comment]
Hi @artembrizinskij , @RFlipper ,
I have just released release 1.9.0 which allows to include row and cell references even when not redundant.
Please feel free to check whether that solves your issue.
Thanks,
   Salvo

**Comment by salvois** [comment]
Hi @artembrizinskij , @RFlipper ,
I'm closing this issue. Should the problem persist, please reopen it.
Thanks,
   Salvo
